### PR TITLE
Permissions

### LINKED ISSUE #290: ghost setup fails with EACCES
Command: `ghost setup --url=https://my-blog.com --db=mysql --dbhost=localhost --dbuser=ghost --dbpass=xxx --dbname=ghost --mail=SMTP --mailservice=Mailgun --mailuser=xxx --mailpass=xxx --sslemail=xxx --no-prompt --start`

This should skip all prompts, choosing to do all the default things and using the completed config which is provided.

However, it fails on the nginx setup step with:

` EACCES: permission denied, mkdir '/var/www/ghost/system'`

I have never seen this before, so must be to do with the new user system.

I think it _may_ be because I already had a ghost user and group on the box? I will now run a test without that. 

Full output:

```
"\u2714 Configuring Ghost",
"\u001b[?25l[08:01:03] Running setup checks [started]",
"[08:01:03] System Stack [started]",
"[08:01:04] System Stack [completed]",
"[08:01:04] Running setup checks [completed]",
"[08:01:04] Setting up instance [started]",
"[08:01:04] Setting up instance [completed]",
"[08:01:04] Setting up mysql [started]",
"MySQL user is not root, skipping additional user setup",
"[08:01:04] Setting up mysql [skipped]",
"[08:01:04] Setting up linux-user [started]",
"[08:01:04] Creating ghost system user [started]",
"[08:01:04] Creating ghost system user [skipped]",
"[08:01:04] Changing directory permissions [started]",
"Running sudo command: chown -R ghost:ghost /var/www/ghost",
"[08:01:04] Changing directory permissions [completed]",
"[08:01:04] Adding current user to ghost group [started]",
"[08:01:04] Adding current user to ghost group [skipped]",
"[08:01:04] Setting up linux-user [completed]",
"[08:01:04] Setting up nginx [started]",
"[08:01:04] Setting up nginx [failed]",
"[08:01:04] \u2192 EACCES: permission denied, mkdir '/var/www/ghost/system'",
"\u001b[?25hDebug Information:",
"    Node Version: v6.11.0",
"    Ghost-CLI Version: 1.0.0-beta.5",
"    Environment: production",
"    Command: 'ghost setup --url=https://my-blog.com --db=mysql --dbhost=localhost --dbuser=ghost --dbpass=xxx --dbname=ghost --mail=SMTP --mailservice=Mailgun --mailuser=xxx --mailpass=xxx --sslemail=xxx --no-prompt --start'",
"",
"Additional log info available in: /home/admin/.ghost/logs/ghost-cli-debug-2017-07-06T08_01_04_871Z.log",
"",
"Please refer to https://docs.ghost.org/docs/installing-ghost-via-the-cli#troubleshooting for troubleshooting.",
"\u001b[?25h"
```

Stack from the log file:

```
Stack: Error: EACCES: permission denied, mkdir '/var/www/ghost/system'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:923:18)
    at mkdirsSync (/usr/lib/node_modules/ghost-cli/node_modules/fs-extra/lib/mkdirs/mkdirs-sync.js:31:9)
    at Object.mkdirsSync (/usr/lib/node_modules/ghost-cli/node_modules/fs-extra/lib/mkdirs/mkdirs-sync.js:37:16)
    at Promise.fromNode.then (/usr/lib/node_modules/ghost-cli/extensions/nginx/index.js:60:16)
    at tryCatcher (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromiseCtx (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/promise.js:606:10)
    at Async._drainQueue (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/async.js:138:12)
    at Async._drainQueues (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/usr/lib/node_modules/ghost-cli/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Code: EACCES
Path: /var/www/ghost/system
```

### PR DESCRIPTION
**Pull request by sebgie**
closes #290
- change ownership of content and versions directory only
- don’t add current user to ghost group

### CODE REVIEW

File: lib/commands/install.js [hunk 1]
```diff
@@ -65,12 +65,12 @@ class InstallCommand extends Command {
...
-                title: 'Summoning Casper',
-                task: this.casper
-            }, {
                 title: 'Linking things',
                 task: this.link.bind(this)
-            }], false, {concurrent: true})
+            }, {
+                title: 'Summoning Casper',
+                task: this.casper
+            }], false, {concurrent: false})
```
Comment (acburdine) [inline lib/commands/install.js]: the `{concurrent: false}` can be removed, as false is the default.

Comment (sebgie) [inline lib/commands/install.js]: ok, current needs to exist so I assumed that would be the correct change.

Comment (acburdine) [inline lib/commands/install.js]: yup 👍

Comment (sebgie) [inline lib/commands/install.js]: updated

**Comment by ErisDS** [comment]
This still creates the same problem I had before, only for the /content/ directory, rather than the whole folder.

The content dir still has drwxr-xr-x permissions, meaning my admin user cannot write to the files inside, and I therefore cannot import images or read the logs.

The message also reads "changing permissions"  still, I think it should be "changing ownership" because permissions are not being changed.

**Pull request merged**